src/3p-frame.js

```diff
@@ -60,18 +60,36 @@ function getFrameAttributes(parentWindow, element, opt_type, opt_context) {
   return attributes;
 }
 
+/** @record */
+class IframeOptions {
+  constructor() {
+    /**
+     * Whether 3p url should not use meta tag.
+     * @type {boolean|undefined}
+     */
+    this.disallowCustom;
+
+    /**
+     * Whether or not elements within the iframe should be fullscreenable.
+     * @type {boolean|undefined}
+     */
+    this.allowFullscreen;
+  }
+}
+
 /**
  * Creates the iframe for the embed. Applies correct size and passes the embed
  * attributes to the frame via JSON inside the fragment.
  * @param {!Window} parentWindow
  * @param {!AmpElement} parentElement
  * @param {string=} opt_type
  * @param {Object=} opt_context
- * @param {boolean=} opt_disallowCustom whether 3p url should not use meta tag.
+ * @param {!IframeOptions=} options Options for the created iframe.
```

Comment:
`opt-options`? sounds redundant but consistent.

Reply:
Changed for consistency. The [google style guide](https://google.github.io/styleguide/jsguide.html#features-functions-default-parameters) mentions they should not be present, but will keep the file consistent. QQ: For new files, should they try to match the style guide or keep with AMP conventions?

Reply:
AMP convention is sometimes-badly-followed Google style guide :) so for new files, stick to the guid